src/OpenTelemetry.Instrumentation.Http/CHANGELOG.md

```diff
@@ -31,7 +31,8 @@
   * [http-metrics](https://github.com/open-telemetry/semantic-conventions/blob/2bad9afad58fbd6b33cc683d1ad1f006e35e4a5d/docs/http/http-metrics.md)
 
 * Added support for publishing `http.client.duration` &
-  `http.client.request.duration` metrics on .NET Framework
+  `http.client.request.duration` metrics on .NET Framework.
+  This supports `HttpClient` and `HttpWebRequest`.
```

Comment:
Seems redundant to me because everything in this library w.r.t. .NET Framework works for both `HttpClient` & `HttpWebRequest`.

https://github.com/open-telemetry/opentelemetry-dotnet/blob/dc1f09d4b968f96955ccfeaa5b0f6f7e9989656a/src/OpenTelemetry.Instrumentation.Http/README.md?plain=1#L6-L12

I would say we should only mention it if it didn't work for one or the other for some reason.

Not a blocker for me, just sayin' 🤔